pyglider/plotting.py

```diff
@@ -223,55 +363,78 @@ def grid_plots(fname, plottingyaml):
         tmean = ds.temperature.mean(axis=1)
         indmax = np.where(~np.isnan(tmean))[0][-1]
         depmax = ds.depth[indmax]
-        fig, axs = plt.subplots(math.ceil(N/2), 2, figsize=(7.5, 7),
+       # fig, axs = plt.subplots(math.ceil(N/2), 2, figsize=(7.5, 7),  #jpnote changed
+        #                        sharex=True, sharey=True)
+        fig, axs = plt.subplots(int(N / 2), 2, figsize=(7.5, 7),
                                 sharex=True, sharey=True)
         axs = axs.flat
         for n, k in enumerate(keys):
             _log.debug(f'key {k}')
 
             pconf = config['pcolor']['vars'][k]
             _log.debug(pconf)
-            cmap = pconf.get('cmap', 'viridis')
+
+            cmap = plt.cm.get_cmap('viridis') #jp
```

Comment:
This change makes the colormap un-configurable...

Reply:
Reverted to the original line.